openapi/components/schemas/StorefrontQuoteReactivateOrder.yaml

```
@@ -0,0 +1,481 @@
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            trialPeriod:
+              description: Date interval of the invoice item trial.
+              type:
+                - 'string'
+                - 'null'
+            taxAmount:
+              description: Invoice item tax.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            setupTaxAmount:
+              description: Tax amount of the setup that is applied to the invoice.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            trialTaxAmount:
+              description: Tax amount of the trial that is applied to the invoice.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+  paymentTerms:
+    description: Payment terms for the customer which are displayed on the quote.
+    type: string
+  expirationTime:
+    description: |-
+      Date and time when the quote expires. The default expiration time is one month from the time the quote is issued.
+      In a `draft` state, this field may be `null`.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+  issuedTime:
+    description: Date and time when the quote is issued.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  acceptedTime:
+    description: Date and time when the quote is accepted.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  rejectedTime:
+    description: Date and time when the quote is rejected.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  canceledTime:
+    description: Date and time when the quote is canceled.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  redirectUrl:
+    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
+    type: string
+    format: uri
+  signature:
+    type: object
+    properties:
+      showWrittenSignatureLines:
+        type: boolean
+        default: false
+        description: Specifies whether to show written signature lines.
+      organizationPrintedName:
+        type:
+          - 'string'
+          - 'null'
+        description: Printed name of the organization.
+        default: null
+  shipping:
+    $ref: ./Shipping.yaml
+  tax:
+    $ref: ./Taxes.yaml
+  _links:
+    type: array
+    description: Related links.
+    readOnly: true
+    items:
+      type: object
+      properties:
+        href:
+          description: Link URL.
+          type: string
+        rel:
+          description: |-
+            Type of link. If quote is accepted, a customer is redirected to `invoicePaymentFormUrl`
+            to pay the initial invoice using one of the methods which are available to the customer.
```

suggestion: for readability
```suggestion
            Type of link. If the quote is accepted, a customer is redirected to the  `invoicePaymentFormUrl` value
            to pay the initial invoice using one of the methods which are available to the customer.
```